appinventor/appengine/src/com/google/appinventor/client/Ode.java

```diff
@@ -186,6 +186,14 @@ public class Ode implements EntryPoint {
 
   private Random random = new Random(); // For generating random nonce
 
+  // Wether or not to show the "Delete Account" Menu public versions
+  // may want this while those managed by a "portal" or other
+  // management system may not want it It is set by the
+  // auth.deleteaccountallowed flag. The default is true.
```

Comment:
Not terribly important, but "Wether" => "Whether" and there appears to be a missing period on line 191.

Reply:
This comment is gone along with the deleteAccountAllowed variable